interfaces/DefaultWidget.ts

```diff
@@ -0,0 +1,559 @@
+			candidate_labels: "space & cosmos, scientific discovery, microbiology, robots, archeology",
+			multi_class: true,
+		}
+	] ],
+	[ "translation", [
+		`My name is Wolfgang and I live in Berlin`,
+		`My name is Sarah and I live in London`,
+	] ],
+	[ "summarization", [
+		`The tower is 324 metres (1,063 ft) tall, about the same height as an 81-storey building, and the tallest structure in Paris. Its base is square, measuring 125 metres (410 ft) on each side. During its construction, the Eiffel Tower surpassed the Washington Monument to become the tallest man-made structure in the world, a title it held for 41 years until the Chrysler Building in New York City was finished in 1930. It was the first structure to reach a height of 300 metres. Due to the addition of a broadcasting aerial at the top of the tower in 1957, it is now taller than the Chrysler Building by 5.2 metres (17 ft). Excluding transmitters, the Eiffel Tower is the second tallest free-standing structure in France after the Millau Viaduct.`,
+	] ],
+	[ "conversational", [
+		`Hey my name is Julien! How are you?`,
+		`Hey my name is Thomas! How are you?`,
+		`Hey my name is Mariama! How are you?`,
+		`Hey my name is Clara! How are you?`,
+		`Hey my name is Julien! How are you?`,
+		`Hi.`,
+	] ],
+	[ "text-generation", [
+		`My name is Julien and I like to`,
+		`My name is Thomas and my main`,
+		`My name is Mariama, my favorite`,
+		`My name is Clara and I am`,
+		`Once upon a time,`,
+	] ],
+	[ "fill-mask", [
+		`Paris is the <mask> of France.`,
+		`The goal of life is <mask>.`,
+	] ],
+]);
+
+const MAPPING_ZH: PerLanguageMapping = new Map([
+	[ "text-classification", [
+		`我喜欢你。 我爱你`,
+	] ],
+	[ "token-classification", [
+		`我叫沃尔夫冈，我住在柏林。`,
+		`我叫萨拉，我住在伦敦。`,
+		`我叫克拉拉，我住在加州伯克利。`,
+	] ],
+	[ "question-answering", [
+		{
+			text: `我住在哪里？`,
+			context: `我叫沃尔夫冈，我住在柏林。`,
+		},
+		{
+			text: `我住在哪里？`,
+			context: `我叫萨拉，我住在伦敦。`,
+		},
+		{
+			text: `我的名字是什么？`,
+			context: `我叫克拉拉，我住在伯克利。`,
+		},
+	] ],
+	[ "translation", [
+		`我叫沃尔夫冈，我住在柏林。`,
+		`我叫萨拉，我住在伦敦。`,
+	] ],
+	[ "summarization", [
+		`该塔高324米（1063英尺），与一幢81层的建筑物一样高，是巴黎最高的建筑物。 它的底座是方形的，每边长125米（410英尺）。 在建造过程中，艾菲尔铁塔超过了华盛顿纪念碑，成为世界上最高的人造结构，它保持了41年的头衔，直到1930年纽约市的克莱斯勒大楼竣工。这是第一个到达300米高度的结构。 由于1957年在塔顶增加了广播天线，因此它现在比克莱斯勒大厦高5.2米（17英尺）。 除发射器外，艾菲尔铁塔是法国第二高的独立式建筑，仅次于米劳高架桥。`,
+	] ],
+	[ "text-generation", [
+		`我叫朱利安，我喜欢`,
+		`我叫托马斯，我的主要`,
+		`我叫玛丽亚，我最喜欢的`,
+		`我叫克拉拉，我是`,
+		`从前，`,
+	] ],
+	[ "fill-mask", [
+		`巴黎是<mask>国的首都。`,
+		`生活的真谛是<mask>。`
+	] ],
+]);
+
+const MAPPING_FR: PerLanguageMapping = new Map([
+	[ "text-classification", [
+		`Je t'apprécie beaucoup. Je t'aime.`,
```

Comment:
This makes me think that these are rarely ever correctly classified. I guess the intent is to have them as two paraphrases as in MRPC, but double sequences like these should have special tokens correctly set and it is not the case as the two sequences are concatenated

Reply:
Yes. Model cards should really specify their own examples, but this worked somewhat when doing NLI using roberta.

not ideal but not sure there's a better solution